### PR TITLE
feature/web

### PR DESCRIPTION
**Pull request by EidrianMax**

### CODE REVIEW

File: staff/adrian-alvarado/web/cv/index.html [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+    <title>Mi curriculum</title>
+  </head>
+  <body>
+    <img
+      src="https://placeimg.com/640/480/people"
+      width="350"
+      alt="Foto Jessica Guillen"
+    />
+    <h1>Jessica Guillen</h1>
+    <h2>Reportera Internacional</h2>
```
Comment (manuelbarzi) [inline staff/adrian-alvarado/web/cv/index.html:12]: may this h2 be a strong or em(phasize) instead?

File: staff/adrian-alvarado/web/cv/index.html [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+
+    <h3>Habilidades</h3>
```
Comment (manuelbarzi) [inline staff/adrian-alvarado/web/cv/index.html:14]: if the above comment is applied, then all these h3 may be changed to h2?

File: staff/adrian-alvarado/web/cv/index.html [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+    <h3>Sobre mí</h3>
+    <p>
+      Lorem, ipsum dolor sit amet consectetur adipisicing elit. Animi accusamus
+      ducimus quis voluptatum dolore iure, alias dolorem repellat. Obcaecati
+      provident minima eveniet deleniti officiis, inventore aperiam error
+      assumenda eius fugit.
+    </p>
+
+    <h3>Contacto</h3>
+    <ol>
```
Comment (manuelbarzi) [inline staff/adrian-alvarado/web/cv/index.html:56]: is this ol required here?

File: staff/adrian-alvarado/web/cv/index.html [hunk 4]
```diff
@@ -0,0 +1,74 @@
...
+      <li>Calle Madrid 1</li>
+      <li>tlf: 91333333333</li>
+      <li><a href="#">Email: jessicaguillen@gmail.com</a></li>
+    </ol>
+
+    <h3>Voluntariados</h3>
+    <p>Lorem ipsum dolor sit amet consectetur adipisicing elit.</p>
+
+    <iframe
```
Comment (manuelbarzi) [inline staff/adrian-alvarado/web/cv/index.html:65]: may you move this self-presentation video up to the "about me" section?

**Comment by EidrianMax** [comment]
Thanks @manuelbarzi changes applied

**Pull request closed without merging**